internal/cli/actions.go

```diff
@@ -310,11 +310,20 @@ func updateActionCmd(cli *cli) *cobra.Command {
 				&inputs.Code,
 				oldAction.GetCode(),
 				inputs.Name+".*.js",
-				cli.actionEditorHint,
```

Comment:
Let's remove `actionEditorHint` as well.

Reply:
This is still being used in the create cmd.